sys/include/net/zep.h

```diff
@@ -7,12 +7,14 @@
  */
 
 /**
- * @defgroup    net_zep  ZEP header definitions
+ * @defgroup    net_zep  ZigBee Encapsulation Protocol
  * @ingroup     net
+ * @brief       Provides definitions for the ZigBee Encapsulation Protocol
```

Comment:
Minor nitpick: Since you removed it in the title (which I didn't ask for) maybe add the abbreviation here in parenthesis after the full name.

Reply:
Done, directly squashed and rebased